vignettes/advanced_data.Rmd

```diff
@@ -0,0 +1,211 @@
+---
+title: "Advanced data"
+output:
+  rmarkdown::html_vignette:
+    fig_width: 7
+    fig_height: 5
+vignette: >
+  %\VignetteIndexEntry{Advanced data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo = F}
+library(knitr)
+knitr::opts_chunk$set(
+  out.extra = 'style="display:block; margin:auto;"'
+)
+```
+
+In the first vignette, we considered only examples where we could observe every chain size exactly, both in theory and in practice.
+This is not always true.
+Some times we only consider chains that have at least some number of cases, and some times we only know that a chain was at least a given size.
+Our investigation of these topics will use data on outbreaks of the pneumonic plague reported by [Nishiura _et al._ (2012)](https://doi.org/10.1016/j.jtbi.2011.10.039).
+
+```{r plague data}
+library(nbbp)
+library(dplyr)
+library(ggplot2)
+data(pneumonic_plague)
+summary(pneumonic_plague)
+```
+
+## Conditioning on observable chain sizes
+
+Among the data, we see a number of relatively small chains, but no singletons.
+This is because when Nishiura _et al._ investigated outbreaks of the pneumonic plague they compiled data only on chains of at least 2 cases.
+
+In general, any particular dataset may include chains observed only of some sizes.
+Analyzing such data requires appropriately conditioning the likelihood.
+This is accomplished using the `condition_geq` argument, which specifies the minimum observation size _for each chain_ (allowing for a mix, if needed).
+For analyzing the pneumonic plague data, we need to set `condition_geq = rep(2, 19)`, indicating that all chains were counted only if they had at least 2 cases.
```

Comment:
In `condition_geq = rep(2, 19)`, what does '19' mean? At first I guessed that chains above 19 cases were also not counted, but this is not right.

Reply:
It means that all 19 chains were only counted if they had size at least 2. The explanation here was definitely a bit convoluted, I've tried to clarify.